account_cryptocurrency/data/res_currency_move_sequence.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
no need encoding

Reply:
Not sure about this.

Reply:
@pedrobaeza do we have a guideline for that?

Reply:
https://github.com/OCA/maintainer-tools/blob/master/template/module/data/model_name_data.xml#L1